package.json

```diff
@@ -45,127 +45,127 @@
-        "when": "editorTextFocus"
+        "when": "editorTextFocus && !vim.debugInput"
       },
       {
         "key": "ctrl+v",
         "command": "extension.vim_ctrl+v",
-        "when": "editorTextFocus && vim.mode != 'Insert Mode' && vim.useCtrlKeys"
+        "when": "editorTextFocus && vim.mode != 'Insert Mode' && vim.useCtrlKeys && !vim.debugInput"
       },
       {
         "key": "ctrl+d",
         "command": "extension.vim_ctrl+d",
-        "when": "editorTextFocus && vim.useCtrlKeys"
+        "when": "editorTextFocus && vim.useCtrlKeys && !vim.debugInput"
       },
       {
         "key": "ctrl+[",
         "command": "extension.vim_ctrl+[",
-        "when": "editorTextFocus && vim.useCtrlKeys"
+        "when": "editorTextFocus && vim.useCtrlKeys && !vim.debugInput"
       },
       {
         "key": "ctrl+w",
         "command": "extension.vim_ctrl+w",
-        "when": "editorTextFocus"
+        "when": "editorTextFocus && !vim.debugInput"
       },
       {
         "key": "ctrl+c",
         "command": "extension.vim_ctrl+c",
-        "when": "editorTextFocus && vim.useCtrlKeys"
+        "when": "editorTextFocus && vim.useCtrlKeys && !vim.debugInput"
       },
       {
         "key": "ctrl+a",
         "command": "extension.vim_ctrl+a",
-        "when": "editorTextFocus && vim.useCtrlKeys"
+        "when": "editorTextFocus && vim.useCtrlKeys && !vim.debugInput"
       },
       {
         "key": "ctrl+x",
         "command": "extension.vim_ctrl+x",
-        "when": "editorTextFocus && vim.useCtrlKeys"
+        "when": "editorTextFocus && vim.useCtrlKeys && !vim.debugInput"
       },
       {
         "key": "ctrl+t",
         "command": "extension.vim_ctrl+t",
-        "when": "editorTextFocus && vim.useCtrlKeys"
+        "when": "editorTextFocus && vim.useCtrlKeys && !vim.debugInput"
       },
       {
         "key": "ctrl+pagedown",
         "command": "extension.vim_ctrl+pagedown",
-        "when": "editorTextFocus && vim.useCtrlKeys"
+        "when": "editorTextFocus && vim.useCtrlKeys && !vim.debugInput"
       },
       {
         "key": "ctrl+pageup",
         "command": "extension.vim_ctrl+pageup",
-        "when": "editorTextFocus && vim.useCtrlKeys"
+        "when": "editorTextFocus && vim.useCtrlKeys && !vim.debugInput"
       },
       {
         "key": "left",
         "command": "extension.vim_left",
-        "when": "editorTextFocus && vim.mode != 'Insert Mode'"
+        "when": "editorTextFocus && vim.mode != 'Insert Mode' && !vim.debugInput"
       },
       {
         "key": "right",
         "command": "extension.vim_right",
-        "when": "editorTextFocus && vim.mode != 'Insert Mode'"
+        "when": "editorTextFocus && vim.mode != 'Insert Mode' && !vim.debugInput"
       },
       {
         "key": "up",
         "command": "extension.vim_up",
-        "when": "editorTextFocus && vim.mode != 'Insert Mode'"
+        "when": "editorTextFocus && vim.mode != 'Insert Mode' && !vim.debugInput"
```

Comment:
Oh wait. Cause these ctrl modifiers are configured they *should* already go through handlekeys right?